Misc/NEWS.d/next/Core and Builtins/2018-08-08-20-52-55.bpo-33989.TkLBui.rst

```diff
@@ -0,0 +1,2 @@
+Ensure that ``ms.key_compare`` is always initialized in :meth:`list.sort`.
```

Comment:
This tells nothing to the end user. `ms.key_compare` is internal name in C sources.

Describe the issue how it can be visible to the end Python or C API user. For example: "Fixed possible crash in list.sort when sort objects with ob_type->tp_richcompare == NULL".